Source/core/FileSystem.cpp

```diff
@@ -56,6 +56,90 @@ namespace Core {
     {
         Close();
     }
+
+    /* static */ string File::Normalize(const string& location, bool& valid)
+    {
+        string result(location);
+
+        valid = true;
```

Comment:
Guess not very important but should empty be valid?

Reply:
Yes, it would indicate the current active path.